x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/without_response_actions_role.json

```diff
@@ -0,0 +1,53 @@
+{
+  "elasticsearch": {
+    "cluster": ["manage"],
+    "indices": [
+      {
+        "names": [
+          ".alerts-security.alerts-default",
+          ".alerts-security.alerts-*",
+          ".siem-signals-*",
+          ".items-*",
+          ".lists-*"
+        ],
+        "privileges": ["all"]
+      }
+    ],
+    "run_as": []
+  },
+  "kibana": [
+    {
+      "base": [],
+      "feature": {
+        "actions": ["all"],
+        "advancedSettings": ["all"],
+        "dev_tools": ["all"],
+        "fleet": ["all"],
+        "generalCases": ["all"],
+        "indexPatterns": ["all"],
+        "osquery": ["all"],
+        "savedObjectsManagement": ["all"],
+        "savedObjectsTagging": ["all"],
+        "siem": [
+          "minimal_all",
+          "endpoint_list_all",
```

Comment:
should this role have this array with only `minimal_all` value?

Reply:
yes, either `minimal_all` or `minimal_read`, otherwise we can't access the `Customize sub-features toggle` to turn off individual response actions, etc.